index.html

```diff
@@ -14,55 +14,129 @@
+                </div>
+                <ul class="socials-container">
+                    <li class="socials">
+                        <a href="https://github.com/Amrit131" class="links"><img src="images/github.png" alt="github link" class="social-img"></a>
+                    </li>
+                    <li class="socials">
+                        <a href="https://www.linkedin.com/in/mishra-amrit/" class="links"><img src="images/linkedin.png" alt="linkedin link" class="social-img"></a>
+                    </li>
+                    <li class="socials">
+                        <a href="http://angellist.com" class="links"><img src="images/dnt.png" alt="dnt link" class="social-img"></a>
+                    </li>
+                    <li class="socials">
+                        <a href="https://twitter.com/i_ammrit" class="links"><img src="images/tweet.png" alt="Twitter link" class="social-img"></a>
+                    </li>
+                    <li class="socials">
+                        <a href="http://medium.com" class="links"><img src="images/microverse.png" alt="Microverse link" class="social-img"></a>
+                    </li>
+                </ul>
+            </div>
+            <div id="bg-container" class="socials-container"> 
+                <img id="bg-img" src="images/Groupbackground.png" alt="Background card">
             </div>
+            <div class="bg-container"></div>
         </section>
         <section class="works" id="my-works">
-            <div id="header2-container">
-                <h2 id="header-2">My Recent Works</h2>
-                <hr id="line">
-            </div>
-            <div id="gymfit-container">
-                <img id="gymfit-img" src="images/gym-fit.png" alt="Gym Fit Image">
-                <h3 id="header-3">Multi-Post Stories</h3>
-                <p id="stories-para">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
-                <ul class="language" id="tech-btn">
-                    <li class="btn"><button>css</button></li>
-                    <li class="btn"><button>html</button></li>
-                    <li class="btn"><button>bootstrap</button></li>
-                    <li class="btn"><button>Ruby</button></li>
-                </ul>
-                <div id="btn-container">
-                    <button id="btn">See Project</button>
+            <div id="works-container">
+                <div id="header2-container">
+                    <h2 id="header-2">My Recent Works</h2>
+                    <hr id="line">
                 </div>
-            </div>
+                <div id="gymfit-container">
+                    <img id="gymfit-img" src="images/gym-fit.png" alt="Gym Fit Image">
```

Comment:
Kindly make sure to follow the exact sizing properties from the Figma design, and fix the width of your image. 

![image](https://user-images.githubusercontent.com/34779928/194727041-b14ae007-b319-4994-8623-c08239917e59.png)

Reply:
# Implemented

![3rd](https://user-images.githubusercontent.com/87594100/194728067-a5d51c01-06f3-4c81-9cc5-a67c6ecc9054.png)